src/components/NcPopover/NcPopover.vue

```diff
@@ -295,7 +492,7 @@ export default {
 		 * @return {HTMLElement|undefined}
 		 */
 		getPopoverTriggerContainerElement() {
-			return this.$refs.popover.$refs.popper.$refs.reference
+			return this.$refs.popover?.$refs.popper.$refs.reference
```

Comment:
Why?

Reply:
Same here

Reply:
It can be undefined when called on mount